opendbc/car/chrysler/interface.py

```diff
@@ -15,7 +15,7 @@ class CarInterface(CarInterfaceBase):
   @staticmethod
   def _get_params(ret: structs.CarParams, candidate, fingerprint, car_fw, alpha_long, docs) -> structs.CarParams:
     ret.brand = "chrysler"
-    ret.dashcamOnly = candidate in RAM_HD
+    ret.dashcamOnly = False  # candidate in RAM_HD  TODO: Remove after testing
```

Comment:
Since upstream does not have Ram HD vehicles removed from dashcam mode, this needs to be reverted after testing.